src/coreclr/tools/Common/TypeSystem/Common/MetadataFieldLayoutAlgorithm.cs

```diff
@@ -369,7 +367,7 @@ protected ComputedInstanceFieldLayout ComputeExplicitFieldLayout(MetadataType ty
                 instanceSize,
                 largestAlignmentRequired,
                 layoutMetadata.Size,
-                alignUpInstanceByteSize: AlignUpInstanceByteSizeForExplicitFieldLayoutCompatQuirk(type),
+                alignUpInstanceByteSize: false,
```

Comment:
Should this rather be `true`? `true` would maintain the existing behavior for blittable and managed sequential types, but allow it to differ for auto layout and similar types.

Reply:
```suggestion
                alignUpInstanceByteSize: true,
```